core/app.go

```diff
@@ -343,6 +349,9 @@ func (a *App) handlePolling() {
 		}
 		a.Telemetry.Serve()
 	}
+	if a.ControlSocket != nil {
```

Comment:
This was definitely neglected during my first pass so I've got the appropriate placing now. I'll also remove the stuff I put in the config parsing (for no reason at all).

Would you also like a CLI flag to override the default FS location of the socket file?